src/access/profiling/parser.py

```diff
@@ -66,3 +73,32 @@ def _convert_from_string(value: str) -> Any:
         except Exception:
             continue
     return value
+
+
+def _read_text_file(file_path: Any) -> str:
+    """Checks whether file_path is a valid path to a text file and tries to read it.
+
+    Args:
+        file_path (Any): the object to check
+
+    Returns:
+        str: The text within the file.
+
+    Raises:
+        TypeError: if file_path is not a valid path
+        FileNotFoundError: if file_path is a path, but is not a file or doesn't exist.
+        ValueError: if file_path is a file, but cannot be read as a text file.
+    """
+
+    try:
+        path = Path(file_path)
```

Comment:
If you pass a missing filename to `Path`, does Path still return a None `path` or does it return `FileNotFoundError`? Specifically, is it possible to catch two exceptions in this try block (instead of the separate `FileNotFoundError` in the next line